### PR TITLE
bla

### PR DESCRIPTION
**Pull request by bensagi**

### CODE REVIEW

File: examples/main.tf [hunk 1]
```diff
@@ -1,14 +1,17 @@
 terraform {
   required_providers {
     securecn = {
-      source = "Portshift/securecn"
+      source  = "Portshift/securecn"
```
Comment (rafaelseidelcisco) [inline examples/main.tf:4]: probably by mistake?

File: securecn/resource_portshift_k8s_cluster.go [hunk 2]
```diff
@@ -92,7 +92,10 @@ func ResourceCluster() *schema.Resource {
 			IstioAlreadyInstalledFieldName:            {Type: schema.TypeBool, Optional: true, Default: false, Description: "if false, istio will be installed, otherwise the controller will use the previously installed istio"},
 			IstioVersionFieldName:                     {Type: schema.TypeString, Optional: true, Default: nil, Computed: true, Description: "if istio already installed, this specifies its version"},
 			IstioIngressEnabledFieldName:              {Type: schema.TypeBool, Optional: true, Computed: true, Description: "If installing Istio, use Istio ingress"},
-			IstioIngressAnnotationsFieldName:          {Type: schema.TypeMap, Elem: schema.TypeString, Optional: true, Default: map[string]string{}, Description: "If enabling Istio ingress, use Istio these ingress annotation"},
+			IstioIngressAnnotationsFieldName:          {Type: schema.TypeMap,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				}, Optional: true, Description: "If enabling Istio ingress, use Istio these ingress annotation"},
```
Comment (rafaelseidelcisco) [inline securecn/resource_portshift_k8s_cluster.go:98]: the text doesn't make sense. maybe "when enabling Istio ingress, use **these Istio** ingress annotations"

File: securecn/resource_portshift_trusted_signer.go [hunk 3]
```diff
@@ -186,8 +186,8 @@ func getTrustedSignerKeysFromConfig(d *schema.ResourceData) []*model2.TrustedSig
 	for k, v := range keysMap {
 		value := v.(string)
 		key := &model2.TrustedSignerKey{
-			Key:  &k,
-			Name: &value,
+			Key:  &value,
```
Comment (rafaelseidelcisco) [inline securecn/resource_portshift_trusted_signer.go:189]: 'name' is the key and 'key' is the value?

Comment (rafaelseidelcisco) [reply securecn/resource_portshift_trusted_signer.go:189]: ok i looked and it looked like you are right..... weird

**Review by rafaelseidelcisco — APPROVED** [review]

File: securecn/resource_portshift_trusted_signer.go (hunk 3 above)
Comment (bensagi) [reply securecn/resource_portshift_trusted_signer.go:189]: yes.
for example: name = key1,
                      key = "-----BEGIN PRIVATE KEY----- asdlkl;a123;olawsd3124" ....

**Pull request merged**